docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -807,6 +799,19 @@ children:
     url: Deploy-for-GDPR-readiness.html
     output: 'web, pdf'
 
+- title: 'Troubleshooting Guide'
```

Comment:
Personally, I would use a shorter title: "Troubleshooting", but I am ok with the longer version too.

If we decide to keep the longer form: Considering there are multiple pages nested under this entry, I think we should use plural (Troubleshooting Guides).

Reply:
I like the shorter title 👍